File: test/java/indent.java
// test: indent_only

class Foo {
  
}

class Foo {
  void bar(
      ) {
    
  }
}

Foo foo = 
    Foo.newBuilder()
    .setBar(Bar.newBuilder()).build();

Foo foo = Foo.newBuilder()
    .setBar(
        Bar
        .newBuilder()
        )
    .build()
    ;

Foo foo = a.
    bar();

Foo foo = a.
    bar(
        );

return (
    ) -> {
  
};

return (
    ) -> 
    1
    ;

MyLambda<String, Long, Object> lambda = 
    (label, value, obj) -> {obj};

MyLambda<String, Long, Object> lambda = (label, 
    value, obj) -> {obj};

MyLambda<String, Long, Object> lambda = (label, value, obj) -> {
  obj};

MyLambda<String, Long, Object> lambda =
    (label, 
        value, obj) -> {obj};

MyLambda<String, Long, Object> lambda =
    (label, value, obj) -> {
  obj};

new int[] {
  0, 1, 2, 3};

new int[] 
    {0, 1, 2, 3};

new int[] {
  0, 
  1, 2, 3};

new int[] {
  0, 1, 2, 3

};
switch (foo) {
  case 1: return b;}

switch (foo) {
  case 1: 
    return b;}

public class Foo {
  @Override
  public Bar bar()
      throws RpcException,
          FooException {
    
  }
}

public class Foo {
  @Override
  public Bar bar(
      )
      throws RpcException, 
          FooException {
    
  }
}

public class Foo {
  @Override
  public Bar bar(
      Baz baz) 
      throws RpcException, FooException {
    
  }
}

public class Foo {
  @Override
  public Bar bar(
      Baz baz)
      throws RpcException, 
          FooException {
    
  }
}

public class Foo {
  @Override
  public Bar bar() 
      throws RpcException, 
          FooException {
    
  }
}

class Foo {
  public void 
      method() {
    
  }
}

class Foo {
  public void
      method() {
    
  }
}

public class Foo 
    extends Bar {
  
}

public class Foo
    extends Bar<
        Baz, Quux> {
  
}

public class Foo
    extends Bar<
        Baz, 
        Quux> {
  
}

foo =
    new Foo() {
      public void bar() {
        
      }
    };

foo =
    new Foo() {
      
      public void bar() {
      }
      
    };

File: test/js/indent.js
// test: indent_only

const 
    foo 
    = 
    1 
    + 
    2
    ;


const foo = 
    [1, 2];

const foo = [
  1, 2];

const foo = 
    {a, b};

const foo = {
  a, b};

someMethod(foo, [
  0, 1, 2,], bar);

someMethod(
    foo, 
    [0, 1, 2,], 
    bar);

someMethod(foo, [
  0, 1, 
  2,
], bar);

someMethod(
    foo,
    [
      1, 2],);

someMethod(
    foo,
    [1, 2],
    );

someMethod(foo, {
  a: 1, b: 2,}, bar);

someMethod(
    foo, 
    {a: 1, b: 2,}, 
    bar);

someMethod(foo, {
  a: 1, b: 2

}, bar);

someMethod(
    foo,
    {
      a: 1, b: 2},);

someMethod(
    foo,
    {a: 1, b: 2},
    );

someMethod(a => 
  a*2
    );

someMethod(a => {
  a*2}
    );

foo()
    .bar(a => a*2);

foo().bar(a => 
  a*2);

foo = 
    function() {
  
};

foo =
    function() {
  
};

switch (foo) {
  case 1: return b;}

switch (foo) {
  case 1: 
    return b;}

class Foo {
  
}

class Foo {
  bar(
      ) {}
}

class Foo {
  bar() {
    
  }
}

if (x) {
  statement;
  
  more;
}
